BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -77,20 +76,26 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
         showRestaurants = restaurants
         showStores = stores
         onlyShowOpenStores = openNow
-        distanceFilter = distanceInMiles
+        distanceFilter = DistanceCalculation().milesToMeters(miles: Double(distanceInMiles))
         
+        if let currentLocation = UserLocation.instance.currentCoordinate {
+            
+            loadStores(at: currentLocation)
+            
+        }
+                
     }
     
     func didCancelFilters() {
         print("onCancel func hit")
+        dismiss(animated: true, completion: nil)
```

Comment:
No, don't dismiss self. The FilterViewController already takes care of closing out the side menu.